docs/csharp/language-reference/operators/index.md

```diff
@@ -75,6 +75,7 @@ The following table lists the C# operators starting with the highest precedence
 | [+x](arithmetic-operators.md#unary-plus-and-minus-operators), [-x](arithmetic-operators.md#unary-plus-and-minus-operators), [\!x](boolean-logical-operators.md#logical-negation-operator-), [~x](bitwise-and-shift-operators.md#bitwise-complement-operator-), [++x](arithmetic-operators.md#increment-operator-), [--x](arithmetic-operators.md#decrement-operator---), [^x](member-access-operators.md#index-from-end-operator-), [(T)x](type-testing-and-cast.md#cast-expression), [await](await.md), [&x](pointer-related-operators.md#address-of-operator-), [*x](pointer-related-operators.md#pointer-indirection-operator-), [true and false](true-false-operators.md) | Unary |
 | [x..y](member-access-operators.md#range-operator-) | Range |
 | [switch](switch-expression.md) | `switch` expression |
+| [with](with-expression.md) | `with` expression |
```

Comment:
This precedence table update is according to the spec proposal: https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/proposals/csharp-9.0/records#with-expression
Is that correct understanding?

Reply:
I'm almost certain this is correct. I'll resolve this conversation.